contrib/scripts/env-setup.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
could you also include an example usage?

Reply:
Good idea, done.